vaadin-combo-box-flow-parent/vaadin-combo-box-flow-integration-tests/src/test/java/com/vaadin/flow/component/combobox/test/RequiredComboboxIT.java

```diff
@@ -76,7 +76,6 @@ private void assertValidationError(ComboBoxElement comboBox) {
                 comboBox.getAttribute("invalid"));
 
         TestBenchElement error = comboBox.$("[part='error-message']").first();
-        Assert.assertTrue(error.getSize().getHeight() > 0);
```

Comment:
At least locally this was flaky, and it seems redundant to the other assertions.